src/cpp/session/modules/NotebookHtmlWidgets.R

```diff
@@ -37,27 +36,28 @@
                           fileext = ".json")
       
       cat(.rs.toJSON(dependencies, unbox = TRUE), file = depfile, sep = "\n")
-
-      # we simulate the viewer pane in the editor surface; create a sizing
-      # policy which fills this space but keeps the widget constrained in
-      # other contexts
-      x$sizingPolicy <- htmlwidgets::sizingPolicy(
-        viewer.padding        = 0,
-        viewer.fill           = TRUE,
-        browser.defaultWidth  = 650,
-        browser.defaultHeight = 400,
-        browser.fill          = FALSE,
-        browser.padding       = 0,
-        knitr.defaultWidth    = 650,
-        knitr.defaultHeight   = 400)
-
-      # save the widget to HTML 
-      htmlwidgets::saveWidget(
-         widget = x, 
-         file = htmlfile,
-         selfcontained = FALSE, 
-         libdir = libraryFolder
-      )
+      
+      # collect knitr options
+      # TODO: parse chunk options and use here?
+      knitrOptions <- knitr::opts_chunk$get()
+      
+      # save as HTML -- save a modified version of the 'standalone' representation
```

Comment:
When displayed within .nb.html the widget should act "as if" it had been knit printed in a regular rmarkdown::render. Does the inclusion of the knitOptions assure this?